### PR TITLE
[#42] [UI] [iOS] As a user, I can see the survey detail screen

### LINKED ISSUE #42: [UI] [iOS] As a user, I can see the survey detail screen
## Why

On the home screen, because of the limitation of the screen size, the user only sees the brief of the survey. 

The survey detail screen is for showing full information about the survey.

## Acceptance criteria

On the survey detail screen, show:

- Full title of the survey
- Full description of the survey
- Back button on the top left corner
- `Start Survey` button in the bottom right corner

When the user taps on the back button, navigate back to the home screen.

When the user taps on the `Start Survey` button, navigate to the first question screen.

## Design

- [Design for survey detail screen](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=34%3A868&scaling=scale-down)

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #42

## What happened 👀

In this PR, I implement the UI for he survey detail screen on iOS:

- Full title of the survey
- Full description of the survey
- Back button on the top left corner
- Start Survey button in the bottom right corner

## Insight 📝

N/A

### CODE REVIEW

File: ios/ios/Resources/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -1 +1,2 @@
 "home.header.today" = "Today";
+"surveyDetail.bottom.start" = "Start Survey";
```
Comment (markgravity) [inline ios/ios/Resources/Localizations/en.lproj/Localizable.strings:2]: What do you think about `"surveyDetail.bottom_button.title"` or `"surveyDetail.primary_button.title"`?

**Review by markgravity — COMMENTED** [review]
The rest LGTM 🙏

**Review by thiennguyen0196 — APPROVED** [review]
Lgtm

**Comment by nmint8m** [comment]
@markgravity Please help me recheck this one. Thank you.

**Review by markgravity — APPROVED** [review]

**Pull request merged**